packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -577,6 +572,21 @@ class _DropdownMenuState<T> extends State<DropdownMenu<T>> {
       };
   }
 
+  // Initialize the text field with the label associated to the entry
+  // whose value matches the initial selection.
+  // Empty the text field when no entry matches the initial selection.
+  void matchInitialSelection() {
+    final int index = filteredEntries.indexWhere((DropdownMenuEntry<T> entry) => entry.value == widget.initialSelection);
```

Comment:
```suggestion
    final int index = filteredEntries.indexWhere(
      (DropdownMenuEntry<T> entry) => entry.value == widget.initialSelection,
    );
```

Reply:
Not fond of this formatting. It reduces the long lines but the style guide explictly mention that going above 80 is preferrable when the result is more readable.
Nonetheless I applied it:
- because readability is somewhat subjective.
- I used the same formatting on a very recent PR.
- maybe we will get autoformat in the coming months (and subjectivity will be gone :smile: ).

Reply:
Not a reviewer but I came across this PR. I'm also not a fan of the indented closure formatting.  How would you both feel about something like:

```dart
TextEditingValue get _initialTextEditingValue {
    for (final entry in filteredEntries) {
      if (entry.value == widget.initialSelection) {
        return TextEditingValue(
          text: entry.label,
          selection: TextSelection.collapsed(offset: entry.label.length),
        );
      }
    }
    return TextEditingValue.empty;
  }
```
where `_localTextEditingController?.value = _initialTextEditingValue` is set in didUpdateWidget. It could also be a function rather than a getter.

I think the current function being a void 'match'-er rather than a setter made the purpose ambigious without viewing the implementation (or the comments).

That said, feel free to ignore this!